### PR TITLE
update the docker tutorial with modern docker

### PR DESCRIPTION
**Pull request by tardyp**
also fixes #3484

### CODE REVIEW

**Comment by tardyp** [comment]
@djmitche I would need access to buildbot org in dockerhub in order to push that docker image

File: master/docs/tutorial/docker.rst [hunk 1]
```diff
@@ -35,6 +31,7 @@ Current Docker dependencies
   For example, Standard Ubuntu, Debian and Arch systems.
 * Packages: lxc, iptables, ca-certificates, and bzip2 packages.
 * Local clock on time or slightly in the future for proper SSL communication.
+* This tutorial uses docker-compose to run a master, a slave, and a postgresql database server
```
Comment (rutsky) [inline master/docs/tutorial/docker.rst]: `worker` here, please

**Comment by tardyp** [comment]
I had to fix the sphinx version to make him happy.

https://github.com/sphinx-doc/sphinx/issues/2673

**Comment by tardyp** [comment]
updated with a version of master built with alpine, and travis testing

**Comment by djmitche** [comment]
> @djmitche I would need access to buildbot org in dockerhub in order to push that docker image

Added

**Comment by seankelly** [comment]
Could you make the new example images less wide? ~1400 pixels is rather large.

**Comment by tardyp** [comment]
Updated with smaller images.

Dockerhub PoC build generated: https://hub.docker.com/r/tardyp/

**Comment by tardyp** [comment]
as there is no objection.. merging to prepare release

**Pull request merged**